Error: Duplicated GUID values

I will work on a path for [this CI Run Error](https://github.com/nodejs/nodejs-news-feeder/actions/runs/6917067033/job/18817812954):

```
Run npm run rss:validate

> nodejs-news-feeder@1.0.0 rss:validate
> node scripts/validate.js


line [4](https://github.com/nodejs/nodejs-news-feeder/actions/runs/6917067033/job/18817812954#step:11:5)[5](https://github.com/nodejs/nodejs-news-feeder/actions/runs/6917067033/job/18817812954#step:11:6), column [8](https://github.com/nodejs/nodejs-news-feeder/actions/runs/6917067033/job/18817812954#step:11:9)3: guid values must not be duplicated within a feed: https://github.com/cutenode/retro-weekly/blob/main/retros/2023-10-22.md [help]... ro-weekly/blob/main/retros/2023-[10](https://github.com/nodejs/nodejs-news-feeder/actions/runs/6917067033/job/18817812954#step:11:11)-22.md</guid>                                             ^

🚨 Feed is invalid!
Error: Process completed with exit code 1.
```

Seems like I introduced this error on #51

--- a/feed.xml
+++ b/feed.xml
@@ -18,8 +18,8 @@
       <title>Retrospective for nodejs from 2023-10-15 to 2023-10-22</title>
       <description><![CDATA[<p>Reporting on 35 Issues from 29 authors, 65 Pull Requests from 33 authors, and 9 Discussions from 7 authors.</p>]]></description>
       <pubDate>Sun, 22 Oct 2023 08:00:00 GMT</pubDate>
-      <link>https://github.com/cutenode/retro-weekly/blob/main/retros/2023-10-22.md</link>
-      <guid>https://github.com/cutenode/retro-weekly/blob/main/retros/2023-10-22.md</guid>
+      <link>https://github.com/cutenode/retro-weekly/blob/main/retros/2023-10-15.md</link>
+      <guid>https://github.com/cutenode/retro-weekly/blob/main/retros/2023-10-15.md</guid>
     </item>
     <item>
       <title>Released nodejs/corepack v0.23.0</title>

